pkg/gencmd/options.go

```diff
@@ -0,0 +1,19 @@
+package gencmd
+
+import (
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+)
+
+type Option func(*Ctx)
+
+func WithGrapiCtx(gctx *grapicmd.Ctx) Option {
+	return func(ctx *Ctx) {
+		ctx.Ctx = gctx
+	}
+}
+
+func WithCreateAppFunc(f CreateAppFunc) Option {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function WithCreateAppFunc should have comment or be unexported